pipelines/rbsa/rbsa.py

```diff
@@ -16,8 +15,11 @@ def __init__(self):
         self.create_tasks()
 
     def create_tasks(self):
-        site_grouping_task = group_sites.SitesGrouper('site_grouping_task')
-        self.pipeline.add_task(site_grouping_task)
+        # site_grouping_task = group_sites.SitesGrouper('site_grouping_task')
+        # self.pipeline.add_task(site_grouping_task)
```

Comment:
Seems like a merge conflict is going to happen. I'm assuming this shouldn't be commented out right?

After you merge this https://github.com/slacgismo/loadinsight/pull/7 in, pull master into this undiscount branch to bring it up to date

Reply:
Yeah, I'll uncomment and pull master into this branch.